Source/FieldSolver/FiniteDifferenceSolver/HybridPICSolveE.cpp

```diff
@@ -697,7 +704,15 @@ void FiniteDifferenceSolver::HybridPICSolveECylindrical (
                 if (solve_for_Faraday) { Ez(i, j, 0) += eta(rho_val, jtot_val) * Jz(i, j, 0); }
 
                 if (include_hyper_resistivity_term) {
-                    auto nabla2Jz = T_Algo::Dzz(Jz, coefs_z, n_coefs_z, i, j, 0, 0);
+                    // r on nodal point (Jz is nodal in r)
+                    Real const r = rmin + i*dr;
+
+                    auto nabla2Jz = 0._rt;
+                    if (r > 0.5_rt*dr) {
+                        nabla2Jz = T_Algo::Dr_rDr_over_r(Jz, r, dr, coefs_r, n_coefs_r, i, j, 0, 0)
+                            + T_Algo::Dzz(Jz, coefs_z, n_coefs_z, i, j, 0, 0);
+                    }
```

Comment:
Do you think the way it is, should be kept, or would this be better as:
```suggestion
                    auto nabla2Jz = T_Algo::Dzz(Jz, coefs_z, n_coefs_z, i, j, 0, 0);
                    if (r > 0.5_rt*dr) {
                        nabla2Jz += T_Algo::Dr_rDr_over_r(Jz, r, dr, coefs_r, n_coefs_r, i, j, 0, 0)
                    }
```

Thereby, essentially just making $r\partial / \partial r \rightarrow 0$ at r = 0.

Reply:
Earlier in this loop there is an exit point if r == 0, so this should never be reached if at origin. Et needs to stay zero at the origin, I removed the check though since it was redundant.